src/test/scala/ai/privado/exporter/PythonLanguageEgressTest.scala

```diff
@@ -74,17 +74,22 @@ class PythonLanguageEgressTest extends PrivadoPySrc2CpgFixture {
       | API_5 =      "api/v2" + "/ce/customers"
       |""".stripMargin)
 
+  override def beforeAll(): Unit = {
+    AppCache.repoLanguage = Language.PYTHON
```

Comment:
This can result in test case flakiness across, this global object needs to be converted to an instance object instead
cc- @nikvin15 @hiteshbedre

Reply:
I took a reference from this file https://github.com/Privado-Inc/privado-core/blob/main/src/test/scala/ai/privado/languageEngine/java/JavaTaggingTestBase.scala#L56. I don't have much idea but is this also not same 🤔 I thought we reset the repoLanguage for every test case.

Reply:
The reference you took is problematic too and also needs to be changed. Usage of global objects often results in overwriting those values in test cases.

Reply:
I see in javascript every suite runs on isolated environment so `AppCache.repoLanguage` will reset for each suite. If that is not the case and I understand you correctly. Could you guide me how to override `AppCache.repoLanguage ` because this is being used directly in the code so I didn't get the instance clue.

Reply:
Synced offline with @khemrajrathore this is a backlog item for privado core so ignoring for now